build.gradle

```diff
@@ -233,9 +234,11 @@ distributions {
 
 docker {
   javaApplication {
-    baseImage = 'openjdk:8-jre-slim'
+    // https://bmuschko.github.io/gradle-docker-plugin/current/user-guide/#extension_2
+    // update base image to openjdk 17 jdk slim. other jdk like termurin, etc generates error
+    baseImage = 'openjdk:17.0.1-jdk-slim'
```

Comment:
It seems the openjdk image is deprecated: https://hub.docker.com/_/openjdk

For instance it did not seem to receive the latest Java 17 updates.

It recommends some alternatives, e.g. [`eclipse-temurin`](https://hub.docker.com/_/eclipse-temurin).

`eclipse-temurin` seems to be based on Ubuntu (so probably compatible to the build?; alpine variant would also be nice if it can be used with low effort), there seems to also be a jre variant.
Generally we should not target specific minor/patch versions but make use of the latest image for the respective Java version (e.g. `17-jre`).

Reply:
> Earlier I tried to run docker plugin with eclipse-temurin 17 image that had a lot of problems. So as per the documentation I used java 11 jre-slim (thinking that docker plugin only supports up to java 11) and it worked.

That's why earlier I tried with temurin. I can check again with `eclipse-temurin:17-jre-alpine`. It works with this image.